koku/api/report/gcp/serializers.py

```diff
@@ -34,12 +34,13 @@
 class GCPGroupBySerializer(GroupSerializer):
     """Serializer for handling query parameter group_by."""
 
-    _opfields = ("account", "region", "service", "project")
+    _opfields = ("account", "region", "service", "project", "instance_type")
```

Comment:
Shouldn't need to be a group by option for GCP, it's just implicit in the instance-type API

Reply:
@adberglund So, I think this is the only place I need to keep the instance_type group at. When I remove it I get:
```
{
    "group_by": {
        "instance_type": [
            "Unsupported parameter or invalid value"
        ]
    }
```